build-linux.sh

```diff
@@ -1,17 +1,34 @@
 #!/bin/bash
 set -e
+set -x
 
 export PATH="$HOME/miniconda/bin:$PATH"
 
-make all
-make python
-mvn clean test -f java/pom.xml
-make test
-make rl_clientlib_test
-cd test
-./test_race_condition.sh
+sudo apt remove --yes --force-yes cmake
+
```

Comment:
We shouldn't be blindly removing cmake. Can we add a "cmake --version" check?

Reply:
We control the image and so we know the cmake version is not current enough(3.5 is still very old) to be used. We would need to extract the version out of `cmake --version`  for that change to be useful and I don't think it is worth the work